src/Cqm/CqmClient.php

```diff
@@ -25,7 +25,13 @@ class CqmClient extends HttpClient
     protected function getCommand(): string
     {
         $port = $this->port;
-        $node = $GLOBALS['node_binary'];
+
+        // The default node binary is 'node', but if the user want's to override it in Globals, they can.
+        // We also set the node binary here to 'node' in case there is no globals option (which also defaults to 'node')
+        $node = 'node';
+        if (!empty($GLOBALS['node_binary'])) {
+            $node = $GLOBALS['node_binary'];
+        }
```

Comment:
If I were stumbling across this, I think I’d like it if the command failed to run, throw an exception, and put a message in exception to fix their path using the globals variable.

Reply:
Sorry but I just don't see a reason to have to have the user intervene with something that can be resolved in code.
While having a node path in globals may be convenient for those whom use different versions of node via NVS, there is no reason for it in production versions.
The node environment and pathing to node binary is setup global on normal node install. The only difference is the binary naming which in the past has been nodejs on ubuntu installs.
In ccda node service start we default to nodejs then revert to node if service fails to start. This has been working pretty well for last couple years.
This may seem minor but I'm having a hard time trying to reconcile the need to have a path in globals.
We can leave it the way you have for now and I'll test further. I do know that it didn't work prior to the fix I suggested in other PR.

Reply:
Okay, I just don't want to have to deal with down the road as I went through this with ccda. I'll research further and add changes as I test. Thanks.